src/clts2vec/parse.py

```diff
@@ -13,11 +13,14 @@ def is_valid_sound(sound):
 
     @note: We expect a string à la "voiced bilabial stop consonant"
     """
```

Comment:
I prefer to do this differently here, try except is not explicit enough, rather add a `isinstance(sound, str) and ( + all the or statements)` to the top. Also less code.